submissions/pseudoscience/pseudoscience_non-fiction.md

```diff
@@ -0,0 +1,107 @@
+## A scientific examination of the Factorio world
+
+Suspension of disbelief is a well-known concept, however that does not stop me from ignoring it completely. Some days ago I saw a [Reddit post](https://www.reddit.com/r/factorio/comments/jaz6yl/calculating_the_density_of_nauvis/) about calculating the density of Nauvis, and that gave me an idea of making a just-for-fun compilation of all the ridiculous science that goes on in the Factorio universe.
+
+Obvious warning: This article contains excessive amounts of math, if you have a crippling math allergy, feel free to skip it.
+
+### Nauvis is a white dwarf
+
+This is a retelling of the Reddit post by [u/DaveMcW](https://www.reddit.com/user/DaveMcW/) that I just mentioned. Note that this assumes Nauvis's day/night cycle is actually caused by rotation, which is something I'll be arguing against later on.
+
+For a planet to not rip itself apart, gravitational acceleration must be greater than centripetal acceleration:
+
+$ g > a $
+
+Using the eqautions for centripetal acceleration and gravitational acceleration, we can show: 
+
+$ GM/r^2 > rw^2 $
+
+Next, we can substitute and rearrange to get an equation for the density of the planet:
+
+$ density > 3π / (T^2G) $
+
+Inputting the values for the constants and time period of Factorio's day/night cycle (T = 25000 ticks = 416.(6) s for anyone curious) we can find:
+
+$ density > 813400 kg/m^2 $
+
+Nauvis's density has to be greater than 813.4 g/cm<sup>3</sup>. That makes Nauvis at least as dense as a white dwarf star. For those of you not in the know, a white dwarf is what's left after a star around the mass of the Sun dies and undergoes collapse, leaving a superheated remnant about the size of the Earth. White dwarves have surface temperatures from 8 000 K to 40 000 K. For comparison, the surface of the Sun is a bit under 6 000 K. Nauvis soil must be an exceptional heat insulator!
```

Comment:
Could we list some other densities here to compare the value to? Maybe like steel and water or something. Might help give a bit of context to the density.